src/redux/news-detail/news-detail.reducer.js

```diff
@@ -0,0 +1,36 @@
+import { SET_ARTICLE } from './news-detail.types';
```

Comment:
I think it is easier to put it inside the `news` reducer. It is better to have it inside one place

Reply:
I think it is better to maintain code this way? because when you delete the page you will not go to another component reduser

Reply:
done